plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -301,6 +301,8 @@
 - python >= 3.5
 
 notes:
+  - Configuring webhook destination with headers start supporting from dnacentersdk 2.9.1 onwards as we are aligned with CatC Release 2.3.7.5.
```

Comment:
Catalyst Center

Reply:
Configuring the webhook destination with headers now supports starting from dnacentersdk version 2.9.1 onwards. This enhancement is in alignment with Catalyst Center Release 2.3.7.5.

Can we write the same for the below as well?

Reply:
updated